src/engine/init.c

```diff
@@ -521,13 +524,20 @@ server_init(int argc, char *argv[])
 	if (rc != 0)
 		goto exit_debug_init;
 
+	rc = dss_engine_metrics_init();
+	if (rc != 0)
+		goto exit_telemetry_init;
+
+	metrics = dss_get_engine_metrics();
+	D_ASSERT(metrics != NULL);
+
 	/** Report timestamp when engine was started */
-	(void)d_tm_record_timestamp(NULL, "started_at");
+	d_tm_record_timestamp(metrics->started_time);
 
 	rc = drpc_init();
 	if (rc != 0) {
 		D_ERROR("Failed to initialize dRPC: "DF_RC"\n", DP_RC(rc));
-		goto exit_telemetry_init;
+		goto exit_metrics_init;
```

Comment:
As for others, this should not be fatal.

Reply:
Fixed.